pkg/apis/devices.harvesterhci.io/v1beta1/node.go

```diff
@@ -0,0 +1,23 @@
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// a Node represents a k8s node and is used to reconcile objects on each node
+type Node struct {
```

Comment:
can u briefly explain why we need the new node CRD?

Reply:
just an internal type to requeue and trigger reconcile of objects. This avoids having to use the infinite loop + ticker logic from earlier. Allows me to use exponential back off to re-trigger reconciles in case of failures.